django/db/models/fields/related.py

```diff
@@ -1741,6 +1778,12 @@ def contribute_to_related_class(self, cls, related):
 
 
 class ForeignKey(ForeignObject):
+    # Field flags
+    one_to_many = True
```

Comment:
I disagree - To my mind, ForeignKeys are One-to-many: they're "one" on the side that defines them, and "many" on the other side. This is backed up by the fact that the "other" side of a ForeignKey is a ManyToOneRel. 

It's worth noting that this same confusion exists in various [Entity-Relationship diagram notations](http://en.wikipedia.org/wiki/Entity–relationship_model#Cardinalities) - the side on which you put the visual "many" varies (c.f. Martin/Crow foot and IDEF1X notation). So, to some extent, this is a bikeshed; my preference would be to stick with the notation we've been using historically.